src/prototype_creep_move.js

```diff
@@ -54,9 +54,8 @@ Creep.prototype.moveToMy = function(target, range=1) {
 
   // Fallback to moveTo when the path is incomplete and the creep is only switching positions
   if (search.path.length < 2 && search.incomplete) {
-    this.creepLog(`moveToMy fallback ${JSON.stringify(target)} ${JSON.stringify(search)}`);
-    this.moveTo(target, {range: range});
-    return false;
+    this.room.debugLog('routing', `moveToMy fallback target: ${JSON.stringify(target)} range: ${range} search: ${JSON.stringify(search)}`);
```

Comment:
`debuglog ` on RoomObjects

Reply:
We have creepLog for creeps so that you can only get the log output for a single creep in a specific room. In this case I wanted it to be enabled for `routing` for debugging purpose.